profiles/common/modules/custom/tmgmt_poetry/inc/tmgmt_poetry.webservice.inc

```diff
@@ -186,158 +212,225 @@ function FPFISPoetryIntegrationRequest($user, $password, $msg) {
+                  '@language Successfully received the translation file.',
+                  ['@language' => $language]
+                );
+
                 // Update the status to executed when we receive a translation.
-                db_update('poetry_status')
-                  ->fields(array('status' => 'Executed'))
-                  ->condition('tjiid', $job_item->tjiid, '=')
-                  ->execute();
+                _tmgmt_poetry_update_item_status($job_item->tjiid, "", "Executed", "");
               }
+
               // Save the file and make it available in the job.
               $name = "JobID" . $job->tjid . '_' . $job->source_language . '_' . $job->target_language;
               $path = 'public://tmgmt_file/' . $name . '.' . $job->getSetting('export_format');
+
               $dirname = dirname($path);
               if (file_prepare_directory($dirname, FILE_CREATE_DIRECTORY)) {
                 $file = file_save_data($content, $path);
                 file_usage_add($file, 'tmgmt_file', 'tmgmt_job', $job->tjid);
-                $message_job->addMessage('Received tanslation can be downloaded <a href="!link">here</a>.', array('!link' => file_create_url($path)));
+                $message_job->addMessage(
+                  'Received tanslation can be downloaded <a href="!link">here</a>.',
+                  ['!link' => file_create_url($path)]
+                );
               }
             }
             catch (Exception $e) {
-              $message_job->addMessage('@language File import failed with the following message: @message', array(
-                '@language' => $language,
-                '@message' => $e->getMessage(),
-              ), 'error');
+              $message_job->addMessage(
+                '@language File import failed with the following message: @message',
+                [
+                  '@language' => $language,
+                  '@message' => $e->getMessage(),
+                ],
+                'error'
+              );
               watchdog_exception('tmgmt_poetry', $e);
             }
           }
         }
       }
     }
 
-    if ($xml->request['type'] == 'status') {
-      watchdog("Poetry: Status message", "Response of type 'status' with remote reference !reference.", array('!reference' => $main_reference));
-    }
-    // Send answer to poetry.
+    // Send answer 'OK' to poetry.
+    watchdog(
+      "tmgmt_poetry",
+      "Send response 'status' with remote reference !reference: !xml",
+      [
+        '!reference' => $main_reference,
+        '!xml' => htmlentities($xml->asXML()),
+      ],
+      WATCHDOG_DEBUG
+    );
+
     $xml = _tmgmt_poetry_generate_answer_xml($job, 'OK', 0, $request);
+
     return $xml->asXML();
   }
   else {
-    watchdog("Poetry: Error message", "Poetry callback failed authentication.");
+    // Send answer 'ERROR' to poetry.
+    watchdog(
+      "tmgmt_poetry",
+      "Poetry callback failed authentication.",
```

Comment:
t() is applied on the watchdog messages when outputting them, e.g. in theme_dblog_message(), see https://api.drupal.org/api/drupal/modules!dblog!dblog.admin.inc/function/theme_dblog_message/7.x. @voidtek is right it should not be used as you suggested @tatarbj.